src/DataTable/README.md

```diff
@@ -385,7 +385,7 @@ See ``dataViewToggleOptions`` props documentation for all supported props.
       <DataTable.TableControlBar />
 
       {/* which kind of body content to show */}
-      { currentView === "card" && <CardView CardComponent={MiyazakiCard} /> }
+      { currentView === "card" && <CardView CardComponent={MiyazakiCard} srcAlt="Card image" /> }
```

Comment:
[clarification] Should the `srcAlt` be added to the `MiyazakiCard` component directly [here](https://github.com/openedx/paragon/blob/master/www/src/components/exampleComponents/MiyazakiCard.jsx#L10)?